MailCore/Models/Draft.swift

```diff
@@ -482,7 +185,120 @@ public class Draft: Object, Decodable, Identifiable, AbstractDraft {
         self.action = action
     }
 
-    public func asUnmanaged() -> UnmanagedDraft {
-        return .toUnmanaged(managedDraft: self)
+    public static func mailTo(subject: String?,
+                              body: String?,
+                              to: [Recipient], cc: [Recipient], bcc: [Recipient]) -> Draft {
```

Comment:
If we put arguments under each other I would do it for all of them.
I see the Aestetics of keeping to:cc:bcc together, so maybe we should wrap in into a `MailRecipients` struct ?